db/migrate/20150623200743_rename_campaign_pages_tags_to_campaign_pages_tag.rb

```diff
@@ -0,0 +1,4 @@
+class RenameCampaignPagesTagsToCampaignPagesTag < ActiveRecord::Migration
+  def change
```

Comment:
I'm guessing this was auto-generated by my IDE when I refactored the `Tags` model into `Tag`. That migration can just be deleted.